doorkeeper-mongodb.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |gem|
   gem.files = Dir['lib/**/*', 'config/**/*', 'MIT-LICENSE', 'Rakefile', 'README.md']
   gem.test_files = Dir['spec/**/*']
 
-  gem.add_dependency 'doorkeeper', '>= 5.0.0', '< 6.0'
+  gem.add_dependency 'doorkeeper', '>= 4.4', '< 5.0'
```

Comment:
Forked from master, but as far as I can tell, master doesn't actually work for doorkeeper 5.0.

This change should work for 4.4 though.